### PR TITLE
add(epp): applications-veeam-vbem-restapi

### PR DESCRIPTION
**Pull request by garnier-quentin**
## Target version (i.e. version that this PR changes)

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] 23.04.x (staging)
- [ ] Cloud (staging)
- [X] Monitoring Connectors (staging)
- [ ] 23.10.x (next)

### CODE REVIEW

**Review by omercier — APPROVED** [review]

File: pp/integrations/plugin-packs/procedures/applications-veeam-vbem-restapi.md [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+| UNIT                            | Select the unit for last execution time threshold. May be 's'for seconds, 'm' for minutes, 'h' for hours, 'd' for days, 'w' for weeks. Default is seconds | s                       |             |
+| FILTERUID                       | Filter jobs by UID                                                                                                                                        |                         |             |
+| FILTERNAME                      | Filter jobs by name                                                                                                                                       |                         |             |
+| FILTERTYPE                      | Filter jobs by type                                                                                                                                       |                         |             |
+| WARNINGEXECUTIONSTATUS          | Set warning threshold for last job execution status (Default: %{status} =~ /warning/i). Can used special variables like: %{status}, %{jobName}            | %{status} =~ /warning/i |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-veeam-vbem-restapi.md:208]:
```suggestion
| WARNINGEXECUTIONSTATUS          | Set warning threshold for last job execution status (Default: %{status} =~ /warning/i). Can use special variables like: %{status}, %{jobName}            | %{status} =~ /warning/i |             |
```

File: pp/integrations/plugin-packs/procedures/applications-veeam-vbem-restapi.md [hunk 2]
```diff
@@ -0,0 +1,399 @@
...
+| CRITICALEXECUTIONSTATUS         | Set critical threshold for last job execution status (Default: %{status} =~ /failed/i). Can used special variables like: %{status}, %{jobName}            | %{status} =~ /failed/i  |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-veeam-vbem-restapi.md:209]:
```suggestion
| CRITICALEXECUTIONSTATUS         | Set critical threshold for last job execution status (Default: %{status} =~ /failed/i). Can use special variables like: %{status}, %{jobName}            | %{status} =~ /failed/i  |             |
```

File: pp/integrations/plugin-packs/procedures/applications-veeam-vbem-restapi.md [hunk 3]
```diff
@@ -0,0 +1,399 @@
...
+4. Fill in the macros you want. Some macros are mandatory.
+
+| Macro               | Description                                                                                           | Default value     | Mandatory   |
+|:--------------------|:------------------------------------------------------------------------------------------------------|:------------------|:------------|
+| VBEMAPIPASSWORD     | Set password                                                                                          |                   |             |
+| VBEMAPIPORT         | Port used                                                                                             | 9398              |             |
+| VBEMAPIPROTOCOL     | Specify https if needed                                                                               | https             |             |
+| VBEMAPIUSERNAME     | Set username                                                                                          |                   |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-veeam-vbem-restapi.md:188]: not mandatory?

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-veeam-vbem-restapi.md [hunk 4]
```diff
@@ -0,0 +1,399 @@
...
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Contenu du pack
+
+### Modèles
+
+Le connecteur de supervision **Veeam Backup Enterprise Manager Rest API** apporte un modèle d'hôte :
+
+* **App-Veeam-Vbem-Restapi**
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-veeam-vbem-restapi.md:14]:
```suggestion
* **App-Veeam-Vbem-Restapi-custom**
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-veeam-vbem-restapi.md [hunk 5]
```diff
@@ -0,0 +1,399 @@
...
+
+Le connecteur apporte les modèles de service suivants
+(classés selon le modèle d'hôte auquel ils sont rattachés) :
+
+<Tabs groupId="sync">
+<TabItem value="App-Veeam-Vbem-Restapi" label="App-Veeam-Vbem-Restapi">
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-veeam-vbem-restapi.md:20]:
```suggestion
<TabItem value="App-Veeam-Vbem-Restapi-custom" label="App-Veeam-Vbem-Restapi-custom">
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-veeam-vbem-restapi.md [hunk 6]
```diff
@@ -0,0 +1,399 @@
...
+
+| Alias        | Modèle de service                   | Description         | Découverte |
+|:-------------|:------------------------------------|:--------------------|:-----------|
+| Repositories | App-Veeam-Vbem-Restapi-Repositories | Contrôle les dépôts | X          |
+
+> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **App-Veeam-Vbem-Restapi** est utilisé.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-veeam-vbem-restapi.md:26]:
```suggestion
> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **App-Veeam-Vbem-Restapi-custom** est utilisé.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-veeam-vbem-restapi.md [hunk 7]
```diff
@@ -0,0 +1,399 @@
...
+4. Renseignez les macros désirées. Attention, certaines macros sont obligatoires.
+
+| Macro               | Description                                                                                           | Valeur par défaut | Obligatoire |
+|:--------------------|:------------------------------------------------------------------------------------------------------|:------------------|:------------|
+| VBEMAPIUSERNAME     | Set username                                                                                          |                   | X           |
+| VBEMAPIPASSWORD     | Set password                                                                                          |                   | X           |
+| VBEMAPIPORT         | Port used                                                                                             | 9398              |             |
+| VBEMAPIPROTOCOL     | Specify https if needed                                                                               | https             |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-veeam-vbem-restapi.md:187]:
```suggestion
| VBEMAPIPROTOCOL     | Specify HTTPS if needed                                                                               | https             |             |
```

File: pp/integrations/plugin-packs/procedures/applications-veeam-vbem-restapi.md [hunk 8]
```diff
@@ -0,0 +1,398 @@
...
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Pack assets
+
+### Templates
+
+The Monitoring Connector **Veeam Backup Enterprise Manager Rest API** brings a host template:
+
+* **App-Veeam-Vbem-Restapi**
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-veeam-vbem-restapi.md:14]:
```suggestion
* **App-Veeam-Vbem-Restapi-custom**
```

File: pp/integrations/plugin-packs/procedures/applications-veeam-vbem-restapi.md [hunk 9]
```diff
@@ -0,0 +1,398 @@
...
+
+The connector brings the following service templates (sorted by the host template they are attached to):
+
+<Tabs groupId="sync">
+<TabItem value="App-Veeam-Vbem-Restapi" label="App-Veeam-Vbem-Restapi">
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-veeam-vbem-restapi.md:19]:
```suggestion
<TabItem value="App-Veeam-Vbem-Restapi-custom" label="App-Veeam-Vbem-Restapi-custom">
```

File: pp/integrations/plugin-packs/procedures/applications-veeam-vbem-restapi.md [hunk 10]
```diff
@@ -0,0 +1,398 @@
...
+
+| Service Alias | Service Template                    | Service Description | Discovery  |
+|:--------------|:------------------------------------|:--------------------|:-----------|
+| Repositories  | App-Veeam-Vbem-Restapi-Repositories | Check repositories  | X          |
+
+> The services listed above are created automatically when the **App-Veeam-Vbem-Restapi** host template is used.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-veeam-vbem-restapi.md:25]:
```suggestion
> The services listed above are created automatically when the **App-Veeam-Vbem-Restapi-custom** host template is used.
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by omercier — APPROVED** [review]

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by omercier — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**